dist/site.webmanifest

```diff
@@ -0,0 +1,8 @@
+{
+    "icons": [{
+        "src": "icon.png",
+        "sizes": "192x192",
+        "type": "image/png"
```

Comment:
You don't need this. The "type" member is intended to be used with more exotic/new types, like WebP, JPEG-XR, etc. So, the intent of "type" is to tell the browser, "hey, if you support this fancy new type, then feel free to use it". As PNG is universally supported, it's redundant to declare it here.